packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -256,6 +256,12 @@ const renderApp = (args) => {
 
     const helmet = Helmet.renderStatic()
 
+    // Remove the stacktrace when executing remotely as to not leak any important
+    // information to users about our system.
+    if (error && isRemote()) {
+        delete error.stack
```

Comment:
nit, it would be nice to keep the `error` object to always have the same object type, i.e. always have the same keys `statue` `message` and `stack`. You can explicitly make the value of `stack` to be `undefined`.

Reply:
It turns out that when the value is serialized that `undefined` values aren't included. So the result would be the same.